config/routes.rb

```diff
@@ -30,6 +30,8 @@
   get "/cart", to: "cart#show"
   delete "/cart", to: "cart#empty"
   delete "/cart/:item_id", to: "cart#remove_item"
+  patch "/cart/:item_id/:quantity", to: "cart#edit_quantity"
```

Comment:
Uses the same route to go to the same action which both increments and decrements